tests/run_code_test.py

```diff
@@ -0,0 +1,84 @@
+"""
+Tests for the ability to run code in both the student and teacher view of the application
+"""
+
+from playwright.sync_api import Page, expect
+import pytest
+
+
+@pytest.fixture(params=["http://localhost:8000", "http://localhost:8000/student.html"])
```

Comment:
I would hope for this fixture to automatically navigate to the appropriate page so that `page.goto` does not need to be called separately in every test.